docs/content/ci/LintWorkerApiWorkaround.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Lint Worker Api Workaround
+type: docs
+---
+
+# Workaround for android lint not using worker api 
+
+[Issue #145235363](https://issuetracker.google.com/issues/145235363)
+
+## Problem
+
```

Comment:
It would be superb to draw a diagram to illustrate the problem

![](https://doc-1oo0-20ng-issuetracker.googleusercontent.com/attachments/lq6b7sjce8vtalcl1k7n3unl1hiq9t0c/usgs4ao7dk61shtlekftd1s3ubgvug20/1582199100000/192718/117148118888620090040/145235363::28315676::192718?dat=AFNnbnkE4GBAezaFQ5W0rKBNImDyjQ3-BYBKZavt19UOB97Ay_cSifsHRi3MpZUzAMdIRBsjD4QKqSojePM7xXc1dADOvLkyCYGWANofkDzlecc0BxEEbX5hfe89tjAvU7eMmv2HUhkjs6eruY9LfJ6K3m5jI2xGZoMO6QmKb9vtUTqtP1329dSbU6H0K_zbS76N8RNdcsjE8IA67QnsN3wUsWGMwWHdtjV4MLRCb9-OQ3ItNogW94guVpwOd5Xm-l5P48R7f5MZR1h-UU-zMqWVRR9fH9cAd88DHa7SMdOVn4YTM9EdRqBUf7c3MvOJg8n-yw8lbhFTrPg9TstpM52VVSBolsdQh68yu97W_9zf1Fea8akP1TDrv61zv8mY7EVs2NBmEu_LdCUU8V9M5OoGWrZa1Aj7xRIe-2naWh1n13vvJQHEWeFpjtbgv8KlU3pjDedRALcjiK0w3wApqeI6qkrvLKrsKa9N8vj0ld4QFLkEMgy3UfG81o589v-waIQ0gGPmXNwpXyQur2Gt0BH2epk7TC7L7PDn7GojPlZZU-vVas-uyF2x_g7FIvs5kwmOmj4OaCwsF0MNiJBGhhP7qlt277-gzPpmzGV1WWJu4p-8O98SIEKL2fIcHAE9WpEyJOiqZRcR07T0Z-vVP0sXbB2coLsJsvfSkCknBW_jlpD8dYI0GpY9Iefae3WEkwJuo7hsA1MPnXApMx02H8VzwC4z63hIWQ&u=0&s=1&download=false&nonce=lfjtk39u5ijrq&user=117148118888620090040&hash=u7qrnvr48q24d2tv3i915hh2ldv1m4of)